src/main/java/com/syntaxphoenix/spigot/smoothtimber/SmoothTimber.java

```diff
@@ -21,7 +21,17 @@
 import com.syntaxphoenix.syntaxapi.reflection.ClassCache;
 
 public class SmoothTimber extends JavaPlugin {
+  private static boolean classExists(String name)
```

Comment:
Please ensure that the formatting is the same as we used everywhere else.
Which means
1. 4 spaces in front of the first line (which equals a tab)
2. The first curly bracket on the same line please